helm/templates/rbac/role_binding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    {{- include "etcd-operator.labels" . | nindent 4 }}
+  name: manager-rolebinding
```

Comment:
We could use something like: name: {{ include "etcd-operator.name" . }}-rolebinding